index.bs

```diff
@@ -4132,23 +4139,30 @@ keyword is used, this includes <code class="idl">entries</code>,
 For read–write maplikes, it also includes <code class="idl">clear</code>,
 <code class="idl">delete</code>, and <code class="idl">set</code> methods.
 
-Maplike interfaces must not
-have any [=interface members=]
-named "<code>entries</code>", "<code>forEach</code>",
-"<code>get</code>", "<code>has</code>",
-"<code>keys</code>", "<code>size</code>", or
+Maplike interfaces must not have any [=regular attributes=] or [=constants=] named
+"<code>entries</code>",
+"<code>forEach</code>",
+"<code>get</code>",
+"<code>has</code>",
+"<code>keys</code>",
+"<code>size</code>", or
 "<code>values</code>",
-or have any [=inherited interfaces=]
-that have [=members=] with these names.
-Read–write maplike interfaces must not
-have any [=attributes=]
-or [=constants=] named
-"<code>clear</code>", "<code>delete</code>",
-or "<code>set</code>", or have any [=inherited interfaces=]
-that have [=attributes=] or [=constants=] with these names.
-
-Note: Operations named "<code>clear</code>", "<code>delete</code>",
-or "<code>set</code>" are allowed on read–write maplike
+or have any [=inherited interfaces=] that have [=regular operations=], [=regular attributes=] or
+[=constants=] with these names.
+
+Maplike interfaces must not have any [=regular operations=] or [=constants=] named
+"<code>size</code>",
+or have any [=inherited interfaces=] that have [=regular operations=], [=regular attributes=] or
+[=constants=] with that name.
+
+Read–write maplike interfaces must not have any [=regular attributes=] or [=constants=] named
+"<code>clear</code>",
+"<code>delete</code>", or
+"<code>set</code>",
+or have any [=inherited interfaces=] that have [=regular operations=], [=regular attributes=] or
+[=constants=] with these names.
+
+Note: Operations with these names are allowed on maplike
```

Comment:
Again, looks wrong.  The existing "can be overridden" thing was for methods that might need to be hooked because they modify the map and might need to update other internal data structures.  But we don't really want hooking for random read operations, right?